mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -138,15 +138,21 @@ private void configureExtension(MavenSession session, Configuration configuratio
         final MavenProjectConfigurator mavenProjectConfigurator = new MavenProjectConfigurator(configuration);
         final File dumpedConfigFile = configuration.dump(Paths.get("").toFile());
         session.getAllProjects().forEach(mavenProject -> {
-            mavenProjectConfigurator.configureTestRunner(mavenProject.getModel());
-            copyConfigurationFile(mavenProject.getModel(), dumpedConfigFile);
-            if (isFailedStrategyUsed()) {
-                SurefireReportStorage.copySurefireReports(mavenProject.getModel());
+            MavenPropertyResolver mavenPropertyResolver = new MavenPropertyResolver(mavenProject.getModel());
+            if (mavenPropertyResolver.isSkipTestsSetInPom()) {
+                logger.info("Smart Testing is disabled. Reason: Test Execution has been skipped in %s module.",
```

Comment:
This line is pretty confusing - the ST is not disabled in general - just for the module.

Reply:
That is what I actually wanted it to convey. My thought on this was, if we set the skip test in the pom for a particular module for instance - `core/impl-base` only, then ST should not be disabled for the whole multi-module project in general but only for the particular module for which the tests are skipped.  
WDYT ?

Reply:
> if we set the skip test in the pom for a particular module for instance - core/impl-base only, then ST should not be disabled for the whole multi-module project in general but only for the particular module for which the tests are skipped.

Yes, this is exactly what I would expect. In addition, if the `skipTests` is set in the parent pom file, then it can be still overridden in any of the submodules, so it has to be checked for all modules separately... Also, the log message should correspond to what it is actually doing - just disabling ST for the particular module.
As for the log - on this [line](https://github.com/arquillian/smart-testing/blob/master/mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenProjectConfigurator.java#L35) we log the information that the ST is activated for the module - there we have the information which modules ST is activated for. So here we can log it at `DEBUG` log level, that it is disabled. 
WDYT?